src/scripts/modules/media/body/body.coffee

```diff
@@ -445,11 +445,11 @@ define (require) ->
       # Clear and replace the hash fragment after the content has loaded
       # to force the browser window to find the intended content (as a side effect)
       jumpToHash = () =>
-        if currentPage.get('loaded') and not @fragmentReloaded and window.location.hash
-          @fragmentReloaded = true
-
+        if currentPage.get('loaded') and window.location.hash
           hash = window.location.hash
-          window.location.replace(hash)
+          $pinnable = $('.pinnable')
+          obscured = $pinnable.height() + 50
```

Comment:
How well does this works when the window width is smaller (like on a mobile phone)?

If this number will continue to be used can it be moved into a common file so it is not duplicated?